src/build.py

```diff
@@ -1137,53 +1140,34 @@ def LibCXXABI():
   CopyLibraryToSysroot(os.path.join(SCRIPT_DIR, 'libc++abi.imports'))
 
 
-def Musl():
-  buildbot.Step('musl')
-  build_dir = os.path.join(work_dirs.GetBuild(), 'musl-out')
-  Mkdir(build_dir)
+def Wasi():
+  buildbot.Step('Wasi')
+  build_dir = os.path.join(work_dirs.GetBuild(), 'wasi-sysroot-out')
+  if os.path.isdir(build_dir):
+    Remove(build_dir)
+  cc_env = BuildEnv(build_dir, use_gnuwin32=True)
```

Comment:
Can we drop the `use_gnuwin32=True`? We don't have any "gnuwin32" paths in the WASI sysroot.

Reply:
I think this is so that certain tools can be used by the Makefile?  At least they were need for musl. Maybe even needed for make itself? 

I could try changing this as a followup and see how it effects the windows build.

Reply:
This is for tools used by the musl build script (on the host machine). If it breaks you'll see it on the windows bot.